tasks/collect-data/collect-data.yaml

```diff
@@ -159,8 +160,8 @@ spec:
         RC=0
 
         check_source () { # Expected arguments are [CRD from DISALLOWED_KEYS_JSON, file]
-            for KEY in $(jq -r ".$1[]" <<< $DISALLOWED_KEYS_JSON) ; do
```

Comment:
I suspect that once again, this is because `<<<` is not supported in POSIX `sh`. Use `bash` instead.